packages/paste-website/src/component-examples/RadioButtonGroupExamples.ts

```diff
@@ -0,0 +1,55 @@
+export const TextExample = `const RadioButtonGroupExample = () => {
+  return (
+    <RadioButtonGroup attached name="foo">
```

Comment:
issue: the `legend` prop is required. It's set to required in the `InlineControlGroup` which we extend for the radio button group props. Very few of these examples have a legend provided so when I copy and paste them, I will get a typescript error. Also the props table marks it as optional, which it's not.